common/src/logger.rs

```diff
@@ -103,12 +103,10 @@ impl Logger {
                 if self.x_pos >= self.width() {
                     self.newline();
                 }
-                const BITMAP_LETTER_WIDTH: usize =
-                    get_bitmap_width(FontWeight::Regular, BitmapHeight::Size14);
-                if self.y_pos >= (self.height() - BITMAP_LETTER_WIDTH) {
+                let bitmap_char = get_bitmap(c, FontWeight::Regular, BitmapHeight::Size14).unwrap();
+                if self.y_pos >= (self.height() - bitmap_char.height()) {
```

Comment:
We should also check the width.

Reply:
Ah, you mean in the the `if self.x_pos >= self.width()` check?

Reply:
Yes, unless I'm missing something, that should also be required, shouldn't it?

Reply:
Added in 130d5f0